build_image.py

```diff
@@ -33,6 +33,7 @@ class OfrakImageConfig:
     install_target: InstallTarget
     cache_from: List[str]
     entrypoint: Optional[str]
+    no_forced_buildkit: bool
```

Comment:
Do we need this option? The Dockerstub files expect buildkit -- this process doesn't work without it.

Reply:
@whyitfor this came from Edward's original implementation. No clue whether there are any cases where it might be needed or desired...

Reply:
As we have it, the Docker build process requires buildkit, so using `no_forced_buildkit` will not work. Based on this, I think we should just remove this option -- it's not needed.

@rbs-jacob, is there something here I am missing?

Reply:
As far as I know, we should be able to remove it. 

My suspicion is that this may have been related to getting OFRAK to build with old versions of Docker running on internal blades, but I don't have any specific memories or notes indicating that – just a vibe.

Either way, even if the `DOCKER_BUILDKIT` environment variable is set, on systems too old to have it, that environment variable being set shouldn't break anything.